packages/ui/src/components/Visualization/ContextToolbar/RuntimeSelector/RuntimeSelector.tsx

```diff
@@ -21,7 +21,7 @@ const getIcon = (name: string) => {
         <img src={quarkusLogo} />
       </Icon>
     );
-  } else if (name.includes('SpringBoot')) {
+  } else if (name.replace(' ', '').includes('SpringBoot')) {
```

Comment:
@tplevko, I'm sorry, `replace` only catches the first occurrence :man_facepalming: , my bad. There's also `replaceAll` but is for a newer library, so our safest bet would be to use a RegExp in this case.

First, create a RegExp like the following:
```
const SPACE_REGEX = /\s/g;
```
it could be above `getIcon`:
```

const SPACE_REGEX = /\s/g;
const getIcon = (name: string) => {
  if (name.includes('redhat')) {
...
```

Then use it in the `replace` method.
```suggestion
  } else if (name.replace(SPACE_REGEX, '').includes('SpringBoot')) {
```
It works for old catalogs
![image](https://github.com/user-attachments/assets/b3cb03d4-1e36-42ae-b022-e8217958ed4c)

And also for newer ones generated with your change
![image](https://github.com/user-attachments/assets/77580573-97e5-44d0-9ac7-1204a51ecc3b)